flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksIncrementalCheckpointRescalingTest.java

```diff
@@ -41,14 +41,26 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
 
 /** Tests to guard rescaling from checkpoint. */
+@RunWith(Parameterized.class)
 public class RocksIncrementalCheckpointRescalingTest extends TestLogger {
 
     @Rule public TemporaryFolder rootFolder = new TemporaryFolder();
 
+    @Parameterized.Parameters(name = "useIngestDbRestoreMode: {0}")
+    public static Collection<Boolean> parameters() {
+        return Arrays.asList(false, true);
+    }
+
+    @Parameterized.Parameter public boolean useIngestDbRestoreMode;
```

Comment:
It would be great to parameterise other unit-tests. Maybe `EmbeddedRocksDBStateBackendTest` and `FileStateBackendTest`?

Reply:
Not the FileStateBackend, because this only affects RocksDB. But there are also more rescaling tests that could cover this.